example.py

```diff
@@ -9,3 +9,6 @@ def multiply(a,b):
 
 def divide(a,b): 
   return a / b
+
+def square(a):
```

Comment:
What about `squared`?

```suggestion
def squared(a):
```